src/main/java/com/conveyal/datatools/manager/jobs/NotifyAdminUsersJob.java

```diff
@@ -0,0 +1,49 @@
+package com.conveyal.datatools.manager.jobs;
+
+import com.conveyal.datatools.manager.models.Project;
+import com.conveyal.datatools.manager.persistence.Persistence;
+import com.conveyal.datatools.manager.utils.JobUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.conveyal.datatools.manager.auth.Auth0Users.getVerifiedEmailsForAdmins;
+import static com.conveyal.datatools.manager.utils.NotificationsUtils.sendNotification;
+
+public class NotifyAdminUsersJob extends NotifyUsersJob {
+
+    public static final Logger LOG = LoggerFactory.getLogger(NotifyAdminUsersJob.class);
+    private final String target;
+    private final String message;
+
+    private NotifyAdminUsersJob(String target, String message) {
+        this.target = target;
+        this.message = message;
+    }
+
+    /**
+     * Convenience method to create and schedule a notification job to notify admin users.
+     */
+    public static void createNotification(String target, String message) {
+        if (hasInvalidApplicationURL()) {
+            return;
+        }
+        NotifyAdminUsersJob notifyJob = new NotifyAdminUsersJob(target, message);
+        JobUtils.lightExecutor.execute(notifyJob);
+        LOG.info("Notification job scheduled in light executor");
+    }
```

Comment:
Create a `createNotification(NotifyUsersJob job)` method in the parent class that handles the URL check and the job queuing.

Reply:
Addressed in: https://github.com/ibi-group/datatools-server/pull/616/commits/b38e277e13b2565ec67c994491a31a3229a256c7